lib/galaxy/datatypes/converters/fasta_to_bowtie_color_index_converter.xml

```diff
@@ -1,8 +1,8 @@
-<tool id="CONVERTER_fasta_to_bowtie_color_index" name="Convert FASTA to Bowtie color space Index" version="1.0.0">
+<tool id="CONVERTER_fasta_to_bowtie_color_index" name="Convert FASTA to Bowtie color space Index" version="1.3.1">
     <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
     <!-- Used on the metadata edit page. -->
     <requirements>
-        <requirement type="package">bowtie</requirement>
+        <requirement type="package" version="1.3.1">bowtie</requirement>
```

Comment:
The `converter_tests` GitHub workflow is now failing. According to http://bowtie-bio.sourceforge.net/index.shtml bowtie 1.3.0 "Fully removed colorspace functionality", so I think we need to use 1.2.3 . Not sure how the test worked before.

Reply:
https://github.com/galaxyproject/galaxy/pull/12851